### PR TITLE
[WIP] Add Unit type to Positions

### PR DESCRIPTION
**Pull request by FelixKleinpeter**
The question are : where do we need to add this parameter? Only in Positions? 
Which function will be changed? Other than those from portfolio.py and positions.py?

### CODE REVIEW

**Comment by FelixKleinpeter** [comment]
I created an Enum class for unit type to add in Positions. Should we use this type for other class, as Trade for exemple? Is there specific functions or files I should not modify? Currently I want to modify the position.py and portfolio.py.

File: src/backlight/asset/currency.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from enum import Enum
+
+
+class Currency(Enum):
+    USD = 1
+    JPY = 2
+    EUR = 3
+    GBP = 4
```
Comment (keisuke-umezawa) [inline src/backlight/asset/currency.py]: Please add ccy at least here. https://www.euronextfx.com/listings

 And order should be defined by https://en.wikipedia.org/wiki/Currency_pair#Base_currency .

File: src/backlight/positions/positions.py [hunk 2]
```diff
@@ -23,7 +23,7 @@ class Positions(pd.DataFrame):
         - `principal`: Principal in your bank account at that moment.
     """
 
-    _metadata = ["symbol"]
+    _metadata = ["symbol", "unit"]
```
Comment (keisuke-umezawa) [inline src/backlight/positions/positions.py]: Need to add `MarketData`, `Labels`, `Trades`. maybe.

Comment (keisuke-umezawa) [reply src/backlight/positions/positions.py]: What is the good name for `unit`?
- unit
- currency_unit
- denomination
- currency denomination

https://www.cmegroup.com/education/featured-reports/nikkei-225-spread-opportunities.html

e.g. We sometime says S&P denominated by JPY.

Comment (FelixKleinpeter) [reply src/backlight/positions/positions.py]: In my point of view currency_unit is clear and not too long, but it's possible that I don't see some ambiguous cases. I will go for it and ctrl+f -> replace all if its not the best choice!

Comment (FelixKleinpeter) [reply src/backlight/positions/positions.py]: I will add the parameter to the from_dataframe functions.

**Comment by FelixKleinpeter** [comment]
The modifications are kinda big and will probably imply to change almost every notebooks using backlight. Should we merge?

**Comment by keisuke-umezawa** [comment]
Memo : before merging it, we need to update major version of it, because it will change IF.

**Comment by keisuke-umezawa** [comment]
> The modifications are kinda big and will probably imply to change almost every notebooks using backlight. Should we merge?

We need to think when to merge it. At least, we need to update major version.

File: src/backlight/positions/positions.py (hunk 2 above)
Comment (keisuke-umezawa) [reply src/backlight/positions/positions.py]: Okay!

**Comment by keisuke-umezawa** [comment]
https://github.com/AlpacaDB/backlight/blob/master/src/backlight/__init__.py#L2-L3
Update versions as 1.0.0.

**Pull request merged**